test/ui/LoginTest.ts

```diff
@@ -0,0 +1,186 @@
+import { expect } from "chai";
+import { LoginPage } from "./LoginPage";
+
+
+describe("LoginPage Tests", () => {
+  let loginPage: LoginPage;
+
+  before(async () => {
+    loginPage = new LoginPage();
+    await loginPage.open();
+  });
+
+  after(async () => {
+    await loginPage.closeBrowser();
+  });
+
+  // vars
+  let adminValidEmail: string = "valid.admin@email.com";
+  let adminValidPassword: string = "admin!Pa$$word123";
+  let applicantValidEmail: string = "regular.user@email.com";
+  let applicantValidPassword: string = "regularU$er123";
+  let invalidEmail: string = "invalid@gmail.com";
+  let invalidPassword: string = "123";
+
+
+  it("should find the email input field", async () => {
+    const emailField = await loginPage.email();
+    expect(emailField).to.not.be.null;
+    expect(await emailField.getAttribute("name")).to.equal("email");
+  });
+
+  it("should find the password input field", async () => {
+    const passwordField = await loginPage.password();
+    expect(passwordField).to.not.be.null;
+    expect(await passwordField.getAttribute("id")).to.equal("password");
+  });
+
+  it("should find the submit button", async () => {
+    const submitButton = await loginPage.submit();
+    expect(submitButton).to.not.be.null;
+    expect(await submitButton.getAttribute("class")).to.include("login-button");
+  });
+
+  // invalid email and invalid password
+  it("should display error message when wrong email and password entered", async () => {
+    const emailField = await loginPage.email();
+    await emailField.clear();
+    await loginPage.enterTextById('email', invalidEmail);
```

Comment:
Maybe we can consolidate these down a bit. Do we need to include a const to capture the email field or can we just call the basePage method for enterTextById() and implement a clear() method into the basepage.